unit/api/application_resource.py

```diff
@@ -71,3 +71,20 @@ def update(self, request: PatchApplicationRequest) -> Union[UnitResponse[Applica
         else:
             return UnitError.from_json_api(response.json())
 
+    def approve_sb(self, request: ApproveApplicationSBRequest):
```

Comment:
This function is fairly specific to our testing suite. I would not worry about pulling this into your master branch